migrations/1669628369773-add-submission-keys.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { alert, error } = require('../src/logger');
+
+const { Schema } = mongoose;
+
+const { connect, close } = require('../src/utils/database');
+
+const Submission = mongoose.model(
+	'submission221128',
+	new mongoose.Schema({
+		grade: { type: Number, min: 0, max: 100 },
+		gradeComment: { type: String },
+		gradeFileIds: [{ type: Schema.Types.ObjectId, ref: 'file' }],
+		submitted: { type: Boolean, default: false },
+		graded: { type: Boolean, default: false },
+	}),
+	'submissions'
+);
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		const resultStepOne = await Submission.updateMany(
+			{},
+			{
+				submitted: true,
+				graded: false,
+			}
+		).exec();
+
+		alert(`Create of key [submitted: true, graded: false] is completely: ${JSON.stringify(resultStepOne)}`);
+
+		const resultStepTwo = await Submission.updateMany(
+			{
+				$or: [
+					{
+						$and: [{ gradeComment: { $exists: true } }, { gradeComment: { $ne: '' } }],
+					},
+					{
+						$and: [{ grade: { $exists: true } }, { grade: { $gte: 0 } }],
```

Comment:
Im legacy wird überprüft nur grade > 0 Aus meine Sicht es ist falsch, da ein Lehrer kann auch 0 % vergeben ?

Reply:
Hatte mich auch schon darüber gewundert. Finde es auch falsch. Wollen wir das mal an UX oder so weitergeben?

Reply:
Ich denke wir können das intern regeln morgen im daily, Cedrick hatte auch in seinem PR auf >= 0 geändert

Reply:
src/services/homework/hooks/index.js habe in dem PR gelöscht